src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -10520,7 +10520,8 @@ void makeCall(SyntaxNode syntax, BoundExpression receiver, MethodSymbol method,
                 indexerOrSliceAccess = BindMethodGroupInvocation(syntax, syntax, method.Name, boundMethodGroup, analyzedArguments,
                     diagnostics, queryClause: null, ignoreNormalFormIfHasValidParamsParameter: true, anyApplicableCandidates: out bool _,
                     disallowExpandedNonArrayParams: false,
-                    acceptOnlyMethods: true).MakeCompilerGenerated(); // Tracked by https://github.com/dotnet/roslyn/issues/76130 : Test effect of acceptOnlyMethods value
+                    acceptOnlyMethods: true) // acceptOnlyMethods is not relevant since we won't search extensions
```

Comment:
>since we won't search extensions [](http://example.com/codeflow?start=82&length=32)

This fact is not obvious. Does this happen because ```boundMethodGroup.SearchExtensions``` is `false`? Consider adding an assert then.